packages/labs/cli/package.json

```diff
@@ -87,7 +87,7 @@
     "command-line-commands": "^3.0.2",
     "command-line-usage": "^6.1.1",
     "tslib": "^2.0.3",
-    "typescript": "~5.1.6"
+    "typescript": "~4.9.0"
```

Comment:
I changed this when trying to resolve unmet peer dependencies a while back. I'll try to change it forward, though I think I'd rather stick with an older version than do a force install, since violated constraints break some npm commands.